### ISSUE TITLE
Calculate deposit options

### ISSUE THREAD

**Issue by saraabejon** [jvinoss/GESPRO_Practica_2_Curso_2024_2025#36]

**Comment by cminguelazamarro** [comment]
¿Puedo calcular opciones de depósito usando la aplicación? ¿Ha habido algún problema?

**Comment by saraabejon** [comment]
> ¿Puedo calcular opciones de depósito usando la aplicación? ¿Ha habido algún problema?

Sí, no ha habido problemas en la implementación de esta acción de la aplicación.
